engines/wintermute/video/video_theora_player.cpp

```diff
@@ -143,7 +143,11 @@ bool VideoTheoraPlayer::initialize(const Common::String &filename, const Common:
 
 	// Additional setup.
 	_surface.create(_theoraDecoder->getWidth(), _theoraDecoder->getHeight(), _theoraDecoder->getPixelFormat());
+#ifdef ENABLE_WME3D
+	_texture = _gameRef->_renderer->createSurface();
```

Comment:
I wonder if there's any reason why the original code isn't using this function, because I get the feeling that is should be doing that. In which case we don't need to special-case for WME_3D, as the renderer should be handling this detail.

Reply:
I couldn't find anything in the player which specifically uses something from BaseSurfaceOSystem and it seems to work with the OpenGL surface anyways. Remove the macro guard?

Reply:
Well, the type of _texture is BaseSurface*, so any subtype details are lost immediately anyhow. The major point of having that factory create-function, was to avoid having to have knowledge of specific renderers outside of the renderer itself.

So I'd say drop the macro-guard, and replace the old code with your call to createSurface(). (BaseRenderOSystem::createSurface performs the exact same "return new BaseSurfaceOSystem(_gameRef);" anyhow.)

While you're at it, you can modify the #include to include "base_surface.h", instead of "base_surface_osystem.h"